containers/vars/defaults.yaml

```diff
@@ -64,5 +68,15 @@ images:
         - "git+https://github.com/pulp/pulp_maven.git"
         - "git+https://github.com/pulp/pulp_python.git"
         - "git+https://github.com/pulp/pulp_rpm.git"
+  - pulp_web_stable:
+      image_name: pulpweb
+      tag: stable
+      container_file: Containerfile.web
+      base_image_name: pulp
+  - pulp_web_master:
+      image_name: pulpweb
```

Comment:
FYI, when building the image I changed it to pulp-web:
https://quay.io/repository/pulp/pulp-web

Reply:
updated.